srcpkgs/applet-window-appmenu/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'applet-window-appmenu'
+pkgname=applet-window-appmenu
+version=0.6.0
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release  .. -Wnodev"
```

Comment:
`-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release  .. ` is all automatically specified by the cmake buildstyle.